spring-security-basic-auth/src/main/java/org/baeldung/security/filter/configuration/CustomWebSecurityConfigurerAdapter.java

```diff
@@ -1,17 +0,0 @@
-package org.baeldung.security.filter.configuration;
-
-import org.baeldung.security.filter.CustomFilter;
-import org.springframework.context.annotation.Configuration;
-import org.springframework.security.config.annotation.web.builders.HttpSecurity;
-import org.springframework.security.config.annotation.web.configuration.WebSecurityConfigurerAdapter;
-import org.springframework.security.web.authentication.www.BasicAuthenticationFilter;
-
-@Configuration
-public class CustomWebSecurityConfigurerAdapter extends WebSecurityConfigurerAdapter {
-
-    @Override
-    protected void configure(HttpSecurity http) throws Exception {
-        http.addFilterAfter(new CustomFilter(), BasicAuthenticationFilter.class);
-    }
-
-}
```

Comment:
The class is being referred in the following article. 
http://www.baeldung.com/spring-security-custom-filter

It might be confusing to reader if we delete the existing class. May be we should just keep the same class and add code here instead.